### DISCUSSION TITLE
Missing array/collection element count in data editor after recent DBeaver update - how to restore?

### DISCUSSION THREAD

**Discussion by vector818** [dbeaver/dbeaver#35264] [Q&A] [answered]
For several versions of DBeaver, the option to display the count of elements in an array/collection within a database record has disappeared (or at least I can’t find it). I'm not 100% sure, but I think the last version that displayed the element count was 24.1.0. Previously, the default display format for arrays was like the one shown in the screenshot below.
![screenshot1](https://github.com/user-attachments/assets/b425903f-7eda-43cf-aa71-cdcc9fb8240e)

Now, it looks like this.
![screenshot2](https://github.com/user-attachments/assets/a33f19f6-12af-4655-b75e-766b3fa60fda)

This specific issue occurs on Windows 11, with the database being PostgreSQL 16.1, and the DBeaver version 24.1.5.202408181605.

### Steps to Reproduce

1. Create a table with the following SQL script:

   ```sql
   CREATE TABLE table_with_array (
       created_at timestamp null,
       date_col date NULL,
       array_col1 _int4 NULL,
       array_col2 _int4 NULL,
       col3 int4 NULL,
       col4 int4 NULL    
   );
   ```

2. Insert data into the table:

```sql
INSERT INTO table_with_array
(created_at, date_col, array_col1, array_col2, col3, col4)
VALUES
    ('2024-08-20 03:00:01.738', '2024-08-20', '{123,128}','{990,995}', 111, 190),
    ('2024-08-20 04:00:01.738', '2024-08-20', '{124,129}','{991,996}', 112, 191),
    ('2024-08-20 05:00:01.738', '2024-08-20', '{125,130}','{992,997}', 113, 192),
    ('2024-08-20 06:00:01.738', '2024-08-20', '{126,131}','{993,998}', 114, 193),
    ('2024-08-20 07:00:01.738', '2024-08-20', '{127,132}','{994,999}', 115, 194);
```
3. Open the created table in the Data viewer.

Is it possible to restore the data presentation format as shown in the first image?

**Comment by uslss** [comment, marked as answer]
will be fixed in 24.2.2
![image](https://github.com/user-attachments/assets/ff98605f-2cb2-4898-8e1a-db07b6f7d671)
can be tested in EA: https://dbeaver.io/files/ea/
